test/renderComponent.test.js

```diff
@@ -19,9 +19,12 @@ var Component = React.createClass({
 describe('#renderComponent', function() {
     it('renders props & children on the component', function () {
         var comp = jsx.renderComponent(Component, { foo: 'bar' }, 'child');
-        
+
         // Verify that it's Off by default
         assert.equal(comp.getDOMNode().textContent, 'child');
         assert.equal(comp.getDOMNode().getAttribute('data-foo'), 'bar');
+
+        jsx.unmountComponent(comp);
```

Comment:
@hankhsiao can you add the test to  `test/unmountComponent.test.js` the idea is so that people can use the tests as documentation to know how to use each method